src/lpython/python_evaluator.cpp

```diff
@@ -47,9 +174,6 @@ Result<std::unique_ptr<LLVMModule>> PythonCompiler::get_llvm3(
     )
 {
 #ifdef HAVE_LFORTRAN_LLVM
-    eval_count++;
-    run_fn = "__lfortran_evaluate_" + std::to_string(eval_count);
-
```

Comment:
We later plan to move the interactivity support into the LibASR so that it is shared between LFortran, LPython and LC. So, it is better that we have/use as common/generic code as possible which easily gets shared across all LCompilers.

Assuming it does not affect the newly added interactivity, I think for now it is better to have it.

Reply:
Can we keep this as part of the next PR? It may require changes in more than one place.

> I was working towards "print the value of the evaluated expression" (https://github.com/lcompilers/lpython/pull/2617#issue-2195119178) goal, and I think there might be some obstacles to doing this. But is doable.

Reply:
>Can we keep this as part of the next PR?

We can do that.